pkg/kudoctl/cmd/package_new.go

```diff
@@ -56,20 +60,45 @@ func newPackageNewCmd(fs afero.Fs, out io.Writer) *cobra.Command {
 	return cmd
 }
 
+func (pkg *packageNewCmd) validateOperatorArg(args []string) error {
+	if pkg.interactive {
```

Comment:
could you just put a comment here that interactive does not need name because we use default name that can be changed or sth like that? It was not obvious to me and I actually had to test it to understand what's going on